test/hooks.test.js

```diff
@@ -664,6 +664,49 @@ test('onRoute hook should preserve handler function in options of shorthand rout
   })
 })
 
+// issue ref https://github.com/fastify/fastify-compress/issues/140
+test('onRoute hook should be called once when prefixTrailingSlash', t => {
```

Comment:
```suggestion
test('onRoute hook should send `prefixing` property when prefixTrailingSlash', t => {
```
Actually, onRoute is called twice.